### PR TITLE
_check_tcpdump() simpler check and unit tests

### PR DESCRIPTION
**Pull request by guedou**
This PR fixes new issues described in PR #2255

### CODE REVIEW

File: scapy/arch/common.py [hunk 1]
```diff
@@ -43,8 +43,7 @@ def _check_tcpdump():
 
     # On some systems, --version does not exist on tcpdump
     return proc.returncode == 0 \
-        or output.startswith(b'Usage: tcpdump ') \
-        or output.startswith(b'tcpdump: unrecognized option')
+        or 'Usage: tcpdump ' in output
```
Comment (p-l-) [inline scapy/arch/common.py]:
```suggestion
        or b'Usage: tcpdump ' in output
```
Or did I miss something?

File: test/regression.uts [hunk 2]
```diff
@@ -6959,6 +6959,55 @@ assert r.linktype == DLT_EN10MB
 l = [ p for p in RawPcapReader(f) ]
 assert len(l) == 1
 
+= _check_tcpdump()
+
+output_ubuntu_19_04 = u"""tcpdump version 4.9.2
```
Comment (p-l-) [inline test/regression.uts]:
```suggestion
output_ubuntu_19_04 = b"""tcpdump version 4.9.2
```

File: test/regression.uts [hunk 3]
```diff
@@ -6959,6 +6959,55 @@ assert r.linktype == DLT_EN10MB
...
+libpcap version 1.8.1
+OpenSSL 1.1.1b  26 Feb 2019
+"""
+returncode_ubuntu_19_04 = 0
+
+output_redhat_4_8 = u"""tcpdump: unrecognized option '--version'
```
Comment (p-l-) [inline test/regression.uts]:
```suggestion
output_redhat_4_8 = b"""tcpdump: unrecognized option '--version'
```

File: test/regression.uts [hunk 4]
```diff
@@ -6959,6 +6959,55 @@ assert r.linktype == DLT_EN10MB
...
+                [ -C file_size ] [ -E algo:secret ] [ -F file ] [ -G seconds ]
+                [ -i interface ] [ -j tstamptype ] [ -M secret ]
+                [ -P in|out|inout ]
+                [ -r file ] [ -s snaplen ] [ -T type ] [ -V file ] [ -w file ]
+                [ -W filecount ] [ -y datalinktype ] [ -z command ]
+                [ -Z user ] [ expression ]
+"""
+returncode_redhat_4_8 = 1
+
+output_centos_6_10 = u"""
```
Comment (p-l-) [inline test/regression.uts]:
```suggestion
output_centos_6_10 = b"""
```

File: scapy/arch/common.py (hunk 1 above)
Comment (guedou) [reply scapy/arch/common.py]: Thanks. I pushed the modified files.

File: test/regression.uts [hunk 5]
```diff
@@ -6959,6 +6959,54 @@ assert r.linktype == DLT_EN10MB
 l = [ p for p in RawPcapReader(f) ]
 assert len(l) == 1
 
+= _check_tcpdump()
+
```
Comment (p-l-) [inline test/regression.uts]:
```suggestion

from scapy.arch.common import _check_tcpdump

```

**Comment by guedou** [comment]
@p-l- ready to be merged.

**Pull request merged**